mantle/build

```diff
@@ -11,7 +11,8 @@ if [[ $# -eq 0 ]]; then
 fi
 
 declare -A BASEARCH_TO_GOARCH=([s390x]=s390x [x86_64]=amd64 [aarch64]=arm64 [ppc64le]=ppc64le)
-KOLET_ARCHES="${KOLET_ARCHES:-s390x x86_64 aarch64 ppc64le}"
+ARCH=$(arch)
+KOLET_ARCHES="${KOLET_ARCHES:-$(ARCH)}"
```

Comment:
Invalid syntax I think? Also feels like we can just keep that as one line.

Reply:
Will fix in next upload. Not sure what you mean by keep it as one line.

I use the `$ARCH` var later in the script.